pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/AggregateOperator.java

```diff
@@ -120,6 +130,7 @@ public String toExplainString() {
 
   @Override
   protected TransferableBlock getNextBlock() {
+    _operatorStats.startTimer();
```

Comment:
so IIUC, this operatorStats is actually operator chain stats, b/c it doesn't stop this timer when calling the downstream's nextBlock() method, yes?

Reply:
it is time spent in getNext call, if we deduct downstream time from this time, we can get the time this operator is spent on processing. 

Are you suggesting we should exclude downstream's time from this?

Reply:
I updated the code to exclude downstream operator time. PTAL

Reply:
sorry just saw this comment. actually I am OK with either approach as long as they are documented properly. but will take another look at the implementation